src/lib.rs

```diff
@@ -33,22 +35,33 @@
 //!     /// unknown data store error
 //!     Unknown,
 //! }
+//!
+//! let error = DataStoreError::Redaction("CLASSIFIED CONTENT".to_string());
+//! assert!("the data for key `CLASSIFIED CONTENT` is not available" == &format!("{}", error));
 //! ```
 //!
 //! <br>
 //!
 //! ## Details
 //!
-//! - A `Display` impl is generated for your type if you provide doc comment
-//!   messages on the struct or each variant of your enum, as shown above in the
-//!   example.
-//!
-//!   The messages support a shorthand for interpolating fields from the error.
-//!
+//! - An `fmt::Display` impl is generated for your enum if you provide
```

Comment:
How are you pronouncing `fmt::Display`? effemtee? I'm surprised by the usage of `an` here, I usually read fmt as format.

Reply:
You correctly divined my internal pronunciation! Thank you for clarifying the canonical form! Fixed!